core/src/main/java/eu/fasten/core/data/JSONUtils.java

```diff
@@ -0,0 +1,287 @@
+     */
+    private static void appendType(StringBuilder result, final String key,
+                                   final JavaType type) {
+        result.append(qoute(key)).append(":{");
+        appendKeyValue(result, "access", type.getAccess());
+        result.append("\"methods\":{");
+        appendMethods(result, type.getMethods());
+        result.append("},\"final\":").append(type.isFinal());
+        result.append(",\"superInterfaces\":[");
+        appendSupers(result, type.getSuperInterfaces());
+        result.append("],");
+        appendKeyValue(result, "sourceFile", type.getSourceFileName());
+        result.append("\"superClasses\":[");
+        appendSupers(result, type.getSuperClasses());
+        result.append("]},");
+
+    }
+
+    /**
+     * Appends information of the super types to the StringBuilder.
+     *
+     * @param supers the list of types to extract the information.
+     * @param result the StringBuilder to append the information.
+     */
+    public static void appendSupers(StringBuilder result, final List<?> supers) {
+        for (final var fastenURI : supers) {
+            result.append(qoute(fastenURI.toString())).append(",");
+        }
+        removeLastIfNotEmpty(result, supers.size());
+
+    }
+
+    /**
+     * Appends methods of a type to the StringBuilder.
+     *
+     * @param methods Map of nodes to extract the information.
+     * @param result the StringBuilder to append the information.
+     */
+    private static void appendMethods(StringBuilder result,
+                                      final Map<Integer, JavaNode> methods) {
+        for (final var entry : methods.entrySet()) {
+
+            result.append(qoute(entry.getKey().toString()));
+            result.append(":{\"metadata\":{");
+            appendMetadata(result, entry.getValue().getMetadata());
+            result.append("},");
+            appendKeyValue(result, "uri", entry.getValue().getUri().toString(), true);
+            result.append("},");
+        }
+        removeLastIfNotEmpty(result, methods.size());
+
+    }
+
+    /**
+     * Appends metadata of a node to the StringBuilder.
+     *
+     * @param map Map of metadata to extract the information.
+     * @param result the StringBuilder to append the information.
+     */
+    private static void appendMetadata(StringBuilder result, final Map<?, ?> map) {
+        for (final var entry : map.entrySet()) {
+            if (entry.getValue() instanceof String) {
+                result.append("\"").append(entry.getKey()).append("\":\"").append(entry.getValue())
+                    .append(
+                        "\",");
+            } else {
+                result.append("\"").append(entry.getKey()).append("\":").append(entry.getValue())
+                    .append(",");
+            }
+        }
+        removeLastIfNotEmpty(result, map.size());
+    }
+
+    /**
+     * Qoutes a given String.
```

Comment:
Typo in the javadoc and in function name

Reply:
fixed 2b8d727